data/zilliqa/assets/zil1gf5vxndx44q6fn025fwdaajnrmgvngdzel0jzp/meta.json

```diff
@@ -1,17 +1,20 @@
 {
-  "name": "BLOX",
-  "symbol": "BLOX",
-  "web": "https://zilliqaroyale.com/",
-  "supply": "5000000",
-  "email": "contact@undefinedcompany.com",
-  "links": {
-    "twitter": "https://twitter.com/zilflip",
-    "telegram": "https://t.me/zilliqaroyale",
-    "linkedin": "https://www.linkedin.com/in/arthur-hoeke-170691103/"
-  },
-  "publicTeam": true,
-  "gen": {
-    "logo": "logo.png",
-    "score": 33
-  }
+    "name": "BLOX",
+    "symbol": "BLOX",
+    "web": "https://blox-sdk.com/",
+    "supply": "5000000",
+    "holders": true,
+    "product": true,
```

Comment:
there's no product, remove

Reply:
We do, tomorrow (Monday) our first product integrating BLOX (ZilliqRoyale.com) launches on mainnet

Reply:
A product means users and adoption, it has to have a minimum of history which you won't have on launch. Have to be patient with that one

Reply:
Updated the PR